src/lib/internal-icons.js

```diff
@@ -839,6 +839,21 @@ const pictogramIcons = {
     viewBox: '0 0 24 24',
     path: `<g fill="none" fill-rule="evenodd"><circle cx="12" cy="12" r="12" fill="#D9F4F3"/><path fill="#00B3B0" fill-rule="nonzero" d="M12.0421023 8.96432868C12.2303137 8.88104932 12.3810493 8.73031369 12.4643287 8.54210234L13.4295364 6.37100953C13.6493939 5.87633016 14.3506061 5.87633016 14.5704636 6.37100953L15.5356713 8.54210234C15.6189507 8.73031369 15.7696863 8.88104932 15.9578977 8.96432868L18.1289905 9.92953641C18.6236698 10.1493939 18.6236698 10.8506061 18.1289905 11.0704636L15.9578977 12.0356713C15.7696863 12.1189507 15.6189507 12.2696863 15.5356713 12.4578977L14.5704636 14.6289905C14.3506061 15.1236698 13.6493939 15.1236698 13.4295364 14.6289905L12.4643287 12.4578977C12.3810493 12.2696863 12.2303137 12.1189507 12.0421023 12.0356713L9.87100953 11.0704636C9.37633016 10.8506061 9.37633016 10.1493939 9.87100953 9.92953641L12.0421023 8.96432868ZM7.97225077 14.9722508 8.51284687 13.3514902C8.66906475 12.8828366 9.33093525 12.8828366 9.48715313 13.3514902L10.0277492 14.9722508 11.6485098 15.5128469C12.1171634 15.6690647 12.1171634 16.3309353 11.6485098 16.4871531L10.0277492 17.0277492 9.48715313 18.6485098C9.33093525 19.1171634 8.66906475 19.1171634 8.51284687 18.6485098L7.97225077 17.0277492 6.35149024 16.4871531C5.88283659 16.3309353 5.88283659 15.6690647 6.35149024 15.5128469L7.97225077 14.9722508Z"/></g>`,
   },
+  'help-pictogram': {
+    viewBox: '0 0 24 24',
+    path: `
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g>
+            <g fill="#D9F4F3">
```

Comment:
this is just a suggestion (this also goes for most of the other icons in this file)  - check if the fill or stroke colors really need to be a statically defined color, or if currentColor would also work. I recently had the problem, that I needed to change the color value for one of the icons. But since this has a background color etc. maybe this is not applicable here and these static colors are fine. :)

Reply:
Makes sense. I will fix this. Thanks!

Reply:
@thomas-storyblok For this case, we need this color as is. It defines the circle background.

Reply:
But I simplified the svg's code, following the other icons. Good catch!

Reply:
@thiagosaife makes sense! I thought that might be the case. Then all is good! 🫡💪🏼🙌🏻